skins/universal/LunnaCat/LunnaCat.skn

```diff
@@ -1104,9 +1104,11 @@ QFrame#EventNotifier {
     padding: 0px;
 }
 
-EventNotifier>QLabel { color: #4BD7FF }
+EventNotifier>QLabel {
+    color: #4BD7FF;
```

Comment:
why?

Reply:
@Vitozz: In order to be consistent everywhere